Automattic/wc-calypso-bridge
# Add task header for "Add a domain" task

### Changes proposed in this Pull Request:



Closes #1008.

Add task header for "Add a domain" task. Note that this change is for all plans.

![Screenshot 2023-03-02 at 12 02 35](https://user-images.githubusercontent.com/4344253/222329382-6597bcfc-f7a6-439e-a079-63a10b4fb187.png)

Figma: lD0gTB4IqGsmSmsnjZNaP6-fi-278%3A10816






### How to test the changes in this Pull Request:




1. Go to `WooCommerce > Home`
2. Complete all tasks except "Add a domain" task
3. Confirm "Add a domain" task has a task header, and it matches the design
4. Click on "Add a domain" button
5. You should be navigated to domain page.



### Other information:




### FOR PR REVIEWER ONLY:
